app/assets/javascripts/discourse/widgets/embedded-post.js.es6

```diff
@@ -27,6 +28,8 @@ export default createWidget('embedded-post', {
   buildKey: attrs => `embedded-post-${attrs.id}`,
 
   html(attrs, state) {
+    this.model = Post.create(attrs);
```

Comment:
could you not call `getAncestorModel` here?

If that doesn't return a post and you have to make one, you should at least use `this.store.createRecord('post', attrs)` instead of importing the `Post` class.